src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/custom.py

```diff
@@ -209,11 +210,37 @@ def get_default_policy(client, scaffold=False):  # pylint: disable=unused-argume
     return _scaffold_certificate_profile() if scaffold else _default_certificate_profile()
 
 
-def create_keyvault(client,
+def recover_keyvault(client, vault_name, resource_group_name, location):
+    """
+    Recover a previously deleted keyvault for which soft delete was enabled
+    :param client:
+    :param vault_name: The name of the vault to recover
+    :param resource_group_name: The original resource group of the vault to recover
+    :param location: The original location of the vault to recover
+    :return: The properties of the recovered key vault
+    """
```

Comment:
This help text should be moved (and reformatted) to the help.py file for the module. This is because the live docs links to help.py if someone wants to make a change to help text.  While this way of creating help text is still supported (for SDK reflection) we don't allow it in custom commands for new PRs.